### DISCUSSION TITLE
API: How to add a issue into organization project board

### DISCUSSION THREAD

**Discussion by Hernrup** [community/community#24526] [Apps, API and Webhooks]
<p>The defined api states</p>
<pre><code>POST /projects/columns/:column_id/cards
</code></pre>
<table><tbody><tr>
<td>content_id</td>
<td>integer</td>
<td>The issue or pull request id you want to associate with this card. You can use the <a href="https://developer.github.com/v3/issues/#list-issues-for-a-repository" target="_blank" rel="nofollow noopener">List issues for a repository</a> and <a href="https://developer.github.com/v3/pulls/#list-pull-requests" target="_blank" rel="nofollow noopener">List pull requests</a> endpoints to find this id.<br><strong>Note:</strong> Depending on whether you use the issue id or pull request id, you will need to specify Issue or PullRequest as the content_type.</td>
</tr></tbody></table>
<p>However this does not work with the issue ids from the repositories. I expect they cant as they are not unique. How do I specify from what repository to to get the issue? As it is defined as an integer it leaves little room for the standard schmea of <code>:org\:repo#:id</code>.</p>

**Comment by hwine** [comment]
<p>Wild guess for workaround: try using the <code>number</code> field (in place of <code>id</code>). I’ve seen huge values there, so I assume they are unique.</p>

**Comment by gwennlbh** [comment, marked as answer]
<p>Using the issue’s id works for me (with PyGithub)</p>
